awsprofile/__init__.py

```diff
@@ -78,9 +80,11 @@ def main():
         else:
             os.putenv('AWS_SESSION_TOKEN', creds.token)
 
-    my_env = os.environ.copy()
-    command_status = os.system(command)
-    exit(os.WEXITSTATUS(command_status))
+    try:
+        subprocess.run(command,
```

Comment:
Unless I'm mistaken, `subprocess.run` wasn't added until python 3.5, is that correct? This project supports back to python 2.6 currently, but I'd like to keep support back to 2.7 (https://github.com/jrstarke/aws-profile/blob/master/setup.py#L51)